packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -339,6 +359,19 @@ export default function ProjectNav({
         </MenuItem>
       </Menu>
 
+      {!isMobile &&
+        admin &&
+        current?.status &&
+        Object.values(ProjectStatus).includes(current?.status) && (
+          <Badge
```

Comment:
We added a lot of misc objects to support this UI, and only this UI. I think it may be better to segregate them all in its own file and just expose a PaymentStatus component we can use.

[We did this in mms as well](https://github.com/10gen/mms/blob/master/client/js/project/common/components/ProjectsLeftNav/ProjectsLeftNavCommon.tsx#L56-L71).

Reply:
👍